docs/containerd-env-setup.md

```diff
@@ -56,28 +56,37 @@ $ sudo tee /etc/nydusd-config.json > /dev/null << EOF
 EOF
 ```
 
-Note:
+Please refer to the nydusd [doc](./nydusd.md) to learn more options.
+
+⚠️ Note:
+
+- The `device.backend.config.scheme` is registry url scheme, leave it empty to automatically detect, otherwise specify to `https` or `http` according to your registry server configuration.
+- The `device.backend.config.auth` is base64 encoded `username:password`, it is required by `nydusd` to lazily pull image data from registry which is authentication enabled, the nydus snapshotter will automatically read docker login auth from the configuration `$HOME/.docker/config.json`, otherwise please copy it to replace `YOUR_LOGIN_AUTH=`.
+- The `device.backend.config.skip_verify` allows to skip insecure registry https certs, only set to `true` if necessary.
+- The `fs_prefetch.enable` enables nydusd to prefetch image data in the background thread, so that container starts faster when it must read a large amount of image data, set to `false` if unneeded.
```

Comment:
- The fs_prefetch.enable option enables nydusd to prefetch image data in a background thread, which can make container startup faster when it needs to read a large amount of image data. You can set this to false if you don't need this functionality.

(powered by ChatGPT)

Reply:
Fixed